src/style.css

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --dark-text: #060b35;
+  --light-text: #616070;
+  --blue-button: #00acdc;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  overflow: hidden;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  width: 100%;
+  height: 36px;
+  margin: 0 16px;
+  max-width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--dark-text);
+}
+
+.card__code {
+  width: 100%;
+  padding: 0;
+  margin: 4px 16px 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--light-text);
+}
+
+.stars {
+  align-self: flex-start;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 16px 0 0 16px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-position: center;
+  background-size: auto;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(2),
+.stars--4 .stars__star:not(:last-child),
+.stars__block .stars--5 .stars__star,
+.stars--3 .stars__star:first-child,
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars__reviews {
+  display: inline-block;
+  font-weight: 400;
+  font-size: 10px;
+  margin-left: 22px;
+}
+
+.card__price {
+  padding: 0;
+  margin: 25px 16px 0;
+  display: flex;
+  justify-content: space-between;
+  width: calc(200px - 34px);
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 16px;
+  color: var(--light-text);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 16px;
+  color: var(--dark-text);
+}
+
+.card__buy-button {
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  cursor: pointer;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  border: 1px solid var(--blue-button);
+  text-align: center;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  -ms-border-radius: 5px;
+  -o-border-radius: 5px;
+  margin: 17px 16px 0;
+}
+
+.card__buy-button:hover {
+  background-color: #fff;
+  border: 1px solid var(--blue-button);
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change just color on :hover

Reply:
fixed